F/flux_core/build_tarballs.jl

```diff
@@ -71,7 +71,7 @@ dependencies = [
     Dependency("Lz4_jll"),
     Dependency("Hwloc_jll"),
     Dependency("SQLite_jll"),
-    Dependency("Lua_jll"),
+    Dependency("Lua_jll"; compat="5.3"),
```

Comment:
If this uses Pkg's version resolution stuff, should this be
```suggestion
    Dependency("Lua_jll"; compat="~5.3"),
```
since otherwise the implicit `^` means that 5.4 would be acceptable?